ansible/templates/etc/nginx/sites-available/svsticky.conf.j2

```diff
@@ -64,6 +65,6 @@ server {
   }
 
   location / {
-    try_files $uri $uri/ /index.php?q=$uri&$args;
+    try_files $uri $uri/ /index.php?q=$uri&$args =404;
```

Comment:
Do we need this? Doesn't WordPress handle 404s itself?

Reply:
You're right, I think I added it because the other template did need it. Removed it again in https://github.com/svsticky/sadserver/commit/15739f01026f73f21633f6732e631a0e1e710155.